bridge/test/contract-mocks/math/MockSigmoid.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT-open-group
+pragma solidity ^0.8.0;
+import "contracts/libraries/math/Sigmoid.sol";
+
+contract MockSigmoid is Sigmoid {
```

Comment:
Can we rename the functions here to fix the solidity linter:
https://github.com/alicenet/alicenet/runs/7441194102?check_suite_focus=true
The solidity linter was being silenced by CI. I just created a PR that will make it fails if the smart contracts don't follow the conventions.